app/views/candidates/index.html.erb

```diff
@@ -8,9 +8,10 @@
     </div>
   </div>
   <% @candidates.each do |candidate|%>
-    <div class="col-sm-3">
-      <%= link_to candidate_path(candidate.id) do %>
-        <div class="card m-2" style="width: 15rem;">
+    <div class="col-sm-3", id = "candidate_card<%= candidate.id %>">
+      <div class="card m-2" style="width: 15rem;">
+        <%= link_to image_tag("delete.png", border: 0, :size => '30x30'), candidate_path(candidate), { method: "delete", remote: true, data: { confirm: "Are you sure?" }, class: "delete_candidate position-absolute top-0 end-0"} %>
```

Comment:
when ever a line gets longer than one frame visibility, break it into few lines.
Example:
 <%= link_to 
  image_tag("delete.png", border: 0, :size => '30x30'), 
candidate_path(candidate), { method: "delete", remote: true, data: { confirm: "Are you sure?" }, class: "delete_candidate position-absolute top-0 end-0"} %>